packaging/debian-unstable/patches/disable-i18n.patch

```diff
@@ -0,0 +1,283 @@
+Description: disable i18n
+ The i18n support in snapd depends on a package that has not yet been packaged.
+ Disable for now until we get the ITP done.
```

Comment:
Can we disable this some other way such as using build tags instead of having the full text of the implementation inverted here?  At the moment it feels like a recipe for pain.

Reply:
Yes, that would probably be a good idea. It would be even better to just get on and package the dependency...